providers/mandrill/src/lib/mandrill.interface.ts

```diff
@@ -10,9 +10,23 @@ export interface IMandrillSendOptions {
     subject: string;
     html: string;
     to: { email: string; type: 'to' | string }[];
+	attachments: IMandrillAttachmentOptions[];	
   };
 }
 
 export interface IMandrillSendResponse {
   _id: string;
 }
+
+export interface IMandrillAttachmentOptions {
+  mime: string;
+  file: Buffer;
+  name?: string;
+  channels?: MandrillChannelTypeEnum[];
+}
+
+export enum MandrillChannelTypeEnum {
+  EMAIL = 'email',
+  SMS = 'sms',
+  DIRECT = 'direct',
```

Comment:
This can be imported after you will merge the code from v0.3.0. No need to implement them again